### ISSUE TITLE
This package versus @axe-core/playwright?

### ISSUE THREAD

**Issue by tstewart15** [abhinaba-ghosh/axe-playwright#215]
Hello, @abhinaba-ghosh !

I discovered your package via the [Storybook docs](https://storybook.js.org/docs/ember/writing-tests/accessibility-testing#automate-accessibility-tests-with-test-runner) and [Blog post](https://storybook.js.org/blog/automate-accessibility-tests-with-storybook/) and am interested in adopting in my team 👍 

I'm curious though about the differences in goals/direction/functionality of this package versus [@axe-core/playwright](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/playwright).

Are both this package and @axe-core/playwright interoperable with Storybook as [documented](https://storybook.js.org/docs/ember/writing-tests/accessibility-testing#automate-accessibility-tests-with-test-runner)?
Is the goal to eventually deprecate this package in favor of @axe-core/playwright?

Perhaps this could be called out in the README.

Thanks!

**Comment by abhinaba-ghosh** [comment]
No. This will always be maintained and new features will be updated. We are internally using it for many usecases that axe doesn't provide out of the box.

**Comment by tstewart15** [comment]
Awesome, thanks! Would you mention this in the README, including the use-cases that this package covers which axe-core's doesn't?
